--- envcontroller.py	2013-06-25 20:14:18.000000000 +0200
+++ envcontroller.py	2013-08-07 02:05:05.420269509 +0200
@@ -118,7 +118,7 @@
             if self.logger:
                 self.logger.Warning("No Respository installed. Reminding user to install it.")
 
-            XbmcWrapper.ShowDialog(LanguageHelper.GetLocalizedString(LanguageHelper.RepoWarningId), LanguageHelper.GetLocalizedString(LanguageHelper.RepoWarningDetailId))
+            #XbmcWrapper.ShowDialog(LanguageHelper.GetLocalizedString(LanguageHelper.RepoWarningId), LanguageHelper.GetLocalizedString(LanguageHelper.RepoWarningDetailId))
 
         return repoAvailable
 
